app/views/accounts/index.html.erb

```diff
@@ -53,15 +53,21 @@
       </tbody>
     </table>
 
-    <h1>Nutzerverwaltung</h1>
+    <h1 id="accounts">Nutzerverwaltung</h1>
```

Comment:
the id string could be a variable at the top of the file and then be used in the links to avoid the same string three times.
this can also be done in the other view files where necessary.

Reply:
Mhh, not sure about that.
The code is not reusable and I don't think this will benefit readability.